src/selfplay/game.cc

```diff
@@ -171,6 +176,15 @@ std::vector<Move> SelfPlayGame::GetMoves() const {
 }
 
 float SelfPlayGame::GetWorstEvalForWinnerOrDraw() const {
+  if (options_[0].uci_options->Get<bool>(kResignWDLStyleId.GetId())) {
```

Comment:
Is it theoretically possible that player1 and player2 have different kResignWDLStyle values set? Then `options_[0]` won't work.

Reply:
Theoretically - yes. But not using the same resign style for both sides seems pretty silly.

Reply:
If it's <=3 lines of code change to add it, would be nice to support it to avoid future surprises.

Reply:
Actually its not.  Its not the final position that matters for this function, its every single position so far. So its a bunch more work to support them having different values.